### DISCUSSION TITLE
Volume Control does not work after the app update

### DISCUSSION THREAD

**Discussion by kbhimani97** [MonitorControl/MonitorControl#1656] [Q&A] [answered]
I use MacBook Pro M1 with my LG 49WQ95C Ultra-wide Monitor. Along with this I Use MX Master Keys as my keyboard. Before the update I use to be able to control my Monitors volume with the dedicated volume buttons on my keyboard but now it does not let me do that neither it allows me to add my own shortcut. 

However I am able to control the brightness with the dedicated keys. The only way to control the volume of my monitor is by using the App shortcut on my menu bar. 

Any Suggestions?

**Comment by waydabber** [comment, marked as answer]
Not sure. You might want to re-add the app to Privacy/Accessibility list under System Settings, maybe that fixes the issue.

**Comment by reeneex70** [comment]
same here with a LG, connected via HDMI. Running the newest version 4.3.3
